src/Compilers/Core/Portable/CodeAnalysis.csproj

```diff
@@ -747,6 +747,7 @@
     <InternalsVisibleToTest Include="Roslyn.Test.Utilities.FX45" />
     <InternalsVisibleToTest Include="Roslyn.Test.Utilities" />
     <InternalsVisibleToTest Include="Roslyn.Compilers.CompilerServer.UnitTests" />
+    <InternalsVisibleToMoq Include="DynamicProxyGenAssembly2" />
```

Comment:
The Moq libraries needs internals visible to in order to for `Mock<T>` declarations to function when `T` is an internal type.